samples/caf_sensor_manager/src/modules/workload_sim.c

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#define MODULE workload_sim
+#include <stdlib.h>
+#include <caf/events/module_state_event.h>
+#include <caf/events/sensor_data_aggregator_event.h>
+
+#define BUSY_TIME CONFIG_WORKLOAD_TIME_PER_SAMPLE
+#define WORKLOAD_SIMULATED_THREAD_STACK_SIZE 800
+#define WORKLOAD_SIMULATED_THREAD_PRIORITY K_LOWEST_APPLICATION_THREAD_PRIO
+#define NUMBER_OF_WORKLOADS DT_PROP(DT_NODELABEL(agg0), buf_count)
+
+LOG_MODULE_REGISTER(MODULE);
+
+struct workload {
+	uint8_t *buf;
+	uint8_t sample_cnt;
```

Comment:
moving sample_cnt close to busy would save some space - now padding needs to be added due to alignment.